docs/pod-metrics.md

```diff
@@ -4,6 +4,7 @@
 | ---------- | ----------- | ----------- | ----------------------- | ----------- | ------ | ------ |
 | kube_pod_annotations | Gauge | Kubernetes annotations converted to Prometheus labels | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `annotation_POD_ANNOTATION`=&lt;POD_ANNOTATION&gt; <br> `uid`=&lt;pod-uid&gt; | EXPERIMENTAL | -
 | kube_pod_info | Gauge | Information about pod | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `host_ip`=&lt;host-ip&gt; <br> `pod_ip`=&lt;pod-ip&gt; <br> `node`=&lt;node-name&gt;<br> `created_by_kind`=&lt;created_by_kind&gt;<br> `created_by_name`=&lt;created_by_name&gt;<br> `uid`=&lt;pod-uid&gt;<br> `priority_class`=&lt;priority_class&gt;<br> `host_network`=&lt;host_network&gt;| STABLE | - |
```

Comment:
For a kube-state-metrics 3.x we should consider removing the pod_ip field here.

Reply:
I agree with this. Would you consider it in scope for this PR?

Reply:
No it shouldn't be in scope for this one. More a remark for other maintainers. :)